compiler/rustc_mir_transform/src/abort_unwinding_calls.rs

```diff
@@ -31,11 +31,7 @@ impl<'tcx> MirPass<'tcx> for AbortUnwindingCalls {
 
         // We don't simplify the MIR of constants at this time because that
         // namely results in a cyclic query when we call `tcx.type_of` below.
-        let is_function = match kind {
-            DefKind::Fn | DefKind::AssocFn | DefKind::Ctor(..) => true,
```

Comment:
This is a change in behavior; `is_fn_like` returns `false` for a `Ctor`.

Reply:
It's fine; ctor won't call other functions.